plugins/Goals/lang/en.json

```
@@ -6,6 +6,8 @@
         "AddNewGoalOrEditExistingGoal": "%1$sAdd a new Goal%2$s or %3$sEdit%4$s existing Goals",
         "AllowGoalConvertedMoreThanOncePerVisit": "Allow Goal to be converted more than once per visit",
         "AllowMultipleConversionsPerVisit": "Allow multiple conversions per visit",
+        "AttributionTitle": "Attribution",
+        "AttributionDocumentation": "Conversion metrics in tables with Goal metrics are attributed to the last non-direct channel that brought visitors who converted.",
```

This might actually not always be true. This only works when using javascript tracking and only if the javascript tracker method `setConversionAttributionFirstReferrer` wasn't called.

---

should we say by default for javascript tracker if that's the case? or it will attribute to first if `setConversionAttributionFirstReferrer` was called?
```suggestion
        "AttributionDocumentation": "Conversion metrics in tables with Goal metrics are attributed to the last non-direct channel, if  `setConversionAttributionFirstReferrer` was called it will attribute to first non-direct channel that brought visitors who converted.",

---

cc: @Stan-vw

---

As conversion attribution is a bit more complexe and might be subject to change I think we should maybe only mention the default case and create and link to a FAQ that documents the behavior and possibilities a bit more detailed.

---

Do we have a FAQ that documents this behaviour?
- if so, I suggest we change the line to "By default, conversion metrics in tables with Goal metrics are attributed to the last non-direct channel that brought visitors who converted. [Learn more](url) about conversion attribution." where the hyperlink goes to the complementary FAQ
- if not, I suggest we change the line to "By default, conversion metrics in tables with Goal metrics are attributed to the last non-direct channel that brought visitors who converted. "

---

@sgiehl do you know if we have a FAQ around that ?

---

I don't know if there is an FAQ. 
But keep in mind, that last non-direct might not be correct in all cases.
When the visitor was visiting the page with campaign parameters in a previous visit, a conversation would be attributed to this campaign even if the conversions visit was coming from another non-direct referer like a search engine.

---

this is the [FAQ](https://matomo.org/faq/general/faq_106/) we have
@sgiehl will this be fine ?
```suggestion
        "AttributionDocumentation": "By default, conversion metrics in tables with Goal metrics are attributed to the last non-direct channel that brought visitors who converted. %1$sLearn  more%2$s about conversion attribution."
```

---

That sounds right to me @AltamashShaikh , thanks for progressing this so rapidly 👍

---

Updated

---

btw The FAQ would probably be more this one: https://matomo.org/faq/general/what-is-the-default-attribution-model-used-in-matomo/

For now I'll move this to a future milestone along with the other relevant: #20516  #20515